config/sql/migration/migrate.5.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN TRANSACTION;
+
+CREATE TABLE rawmeta(director_root BLOB NOT NULL,
+                  director_targets BLOB NOT NULL,
+                  image_root BLOB NOT NULL,
+                  image_targets BLOB NOT NULL,
+                  image_timestamp BLOB NOT NULL,
+                  image_snapshot BLOB NOT NULL);
+
+-- Nothing was stored it root_meta so far, so just drop
+DROP TABLE root_meta;
+CREATE TABLE root_meta(root BLOB NOT NULL, director INTEGER NOT NULL CHECK (director IN (0,1)), version INTEGER NOT NULL);
+CREATE TABLE root_rawmeta(root BLOB NOT NULL, director INTEGER NOT NULL CHECK (director IN (0,1)), version INTEGER NOT NULL);
```

Comment:
I'm always nervous about booleans, they seem to have a habit of collecting more than two values.  What about `repository CHAR NOT NULL CHECK (repository IN ('d','i'))`?

Eric was talking about having a separate TUF repo to handle key rotation, so we might have need for a 3rd type in the future.

Reply:
Makes sense, but INTEGER is also extensible that way. If we want it to be self-descriptive, we might use TEXT field instead.

Reply:
But I'd rather leave everything as is and if we want it to take more than two values eventually, then just add comments to schema.sql about which value means what and rename the field.

Reply:
@cajun-rat That third repo is literally "just tuf" but it serves CA cert bundles. It is completely unrelated to the main Uptane flow.